.pre-commit-config.yaml

```
@@ -59,6 +59,13 @@ repos:
     - id: flake8
       args: ["--count", "--statistics"]
 
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.961
```

```suggestion
    rev: v0.971
```

As [`v0.971`](https://github.com/pre-commit/mirrors-mypy/releases/tag/v0.971) is out now is there a reason to not use it? If so, then we'll need to manually ignore it when pre-commit tries to autoupdate.